### DISCUSSION TITLE
Help with setting up UDR to support youSee IP-TV boxes

### DISCUSSION THREAD

**Discussion by JAagesen** [fabianishere/udm-iptv#194] [Q&A]
Hi I have now changed all my network infrastructure with Ubiquity devices.. U6'es for WI-FI and UDR as router.

I have tried to set up the IGMP Proxy, but there are som details that I don't know, and where youSee in Denmark is not very helpful..

I know that the VLAN used is 101, but using the UDM-iptv setup, it wants the WAN ranges for the provider, and I have no idea of how to get this.. and youSee will not assist.

Anyone out there who can help ??

Thanks in advance
Best regards
Jan

**Comment by bvk007** [comment]
Hi,

You could try with opening up the "internet" and use 0.0.0.0/0 as a CIDR. This could work

**Comment by JAagesen** [comment]
Hi I have tried this now and set up the VLAN to 101, as should be the VLAN the provider uses for IPTV.

The range is set to: 0.0.0.0/0

But my boxes still can’t connect to the IP stream.

Running the diag tool I get the following .

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth4
WAN VLAN: 101 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 0.0.0.0/0
LAN Interfaces: br3
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
=== Service Logs ===
Oct 25 16:46:20 Dream-Router udm-iptv[16215]: udhcpc failed to get a DHCP lease
Oct 25 16:46:20 Dream-Router udm-iptv[16215]: No lease, forking to background
Oct 25 16:46:20 Dream-Router udm-iptv[16215]: NATing IPTV network ranges (if necessary)
Oct 25 16:46:20 Dream-Router udm-iptv[16215]: Setting up igmpproxy
Oct 25 16:46:21 Dream-Router udm-iptv[16215]: Starting igmpproxy
Oct 25 16:46:21 Dream-Router udm-iptv[16215]: adding VIF, Ix 0 Fl 0x0 IP 0x0102a8c0 br3, Threshold: 1, Ratelimit: 0
Oct 25 16:46:21 Dream-Router udm-iptv[16215]: There must be at least 1 Vif as upstream.
Oct 25 16:46:21 Dream-Router systemd[1]: udm-iptv.service: Main process exited, code=exited, status=255/EXCEPTION
Oct 25 16:46:21 Dream-Router systemd[1]: udm-iptv.service: Failed with result 'exit-code'.
Oct 25 16:46:21 Dream-Router systemd[1]: udm-iptv.service: Consumed 385ms CPU time.

Best regards
Jan

**Reply by bvk007** [reply]
You're not getting an IP lease on you IPTV interface...the DHCP options could be in the way...try to remove them or search within the discussions.
